src/browser/upload-file.js

```diff
@@ -7,7 +7,9 @@ function uploadFileBrowser (options = {}) {
   return new Promise((resolve, reject) => {
     const xhr = new window.XMLHttpRequest()
     xhr.open('PUT', options.uploadUrl)
-    xhr.setRequestHeader('Content-Type', options.fileMimeType || '')
```

Comment:
`content-type` should be default because autofilled by browser with boundary

Reply:
We can at least not to set `content-type` in case it was not provided instead of setting it to `''`?